pages/1.11/installing-upgrading/cloud/installing-aws/basic/oss/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: layout.pug
+excerpt:
+title: AWS Basic Installation for DC/OS Open Source
+navigationTitle: AWS Basic Installation for DC/OS Open Source
+menuWeight: 5
+
+oss: true
+---
+
+You can create a DC/OS cluster for Amazon Web Services (AWS) using the <a href="https://downloads.dcos.io/dcos/stable/aws.html" target="_blank">DC/OS templates for AWS CloudFormation</a>.
+
+These instructions provide a basic AWS CloudFormation template that creates a DC/OS cluster that is suitable for demonstrations and POCs. This is the fastest way to get started with the DC/OS templates for AWS CloudFormation.
+
+For a complete set of DC/OS configuration options, see the [Advanced AWS Install Guide](/1.11/installing-upgrading/cloud/installing-aws/advanced/oss/).
+
+**Important:** Upgrades are not supported with this installation method.
```

Comment:
Format this as a warning

Reply:
Content of ENT and OSS are different in Advanced Installer page. So the content is listed sequentially on the same page with separate tags.

Reply:
Content of ENT and OSS are different in Advanced Installer page. So, the content is listed sequentially on the same page with separate tags.